examples/clonable_pcap_dumper.go

```diff
@@ -59,6 +59,9 @@ func (pd pcapdumperParameters) Copy() interface{} {
 	return pdp
 }
 
+func (pd pcapdumperParameters) Remove() {
+}
```

Comment:
File should be closed here

Reply:
Can you please add this in separate commit